ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-cnv-4.14__cnv-odf-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,76 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  ocs-ci-tests:
+    name: ocs-ci-container
+    namespace: ci
+    tag: stable
+  upi-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+images:
+- context_dir: cnv-ci
+  dockerfile_path: Dockerfile
+  to: cnv-ci
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: cnv-odf-tests-aws-upi-ocp414
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
```

Comment:
Don't we need to add this cnv env variable, I saw in last config
`WORKER_INSTANCE_TYPE: c5n.metal`

Reply:
@Simranpal , Earlier the workflow is UPI based so that's why `WORKER_INSTANCE_TYPE` env var is used . But now, we shifted to IPI based workflow i.e. `firewatch-ipi-aws`. So, its not required.

Reply:
Replaced with `COMPUTE_NODE_TYPE`